### PR TITLE
regulators

### PR DESCRIPTION
**Pull request by andrewrynhard**
- feat: enable more power regulators
- test

### CODE REVIEW

File: kernel/kernel/config-arm64 [hunk 1]
```diff
@@ -392,7 +397,7 @@ CONFIG_ARCH_ENABLE_SPLIT_PMD_PTLOCK=y
 CONFIG_SECCOMP=y
 CONFIG_PARAVIRT=y
 # CONFIG_PARAVIRT_TIME_ACCOUNTING is not set
-# CONFIG_KEXEC is not set
+CONFIG_KEXEC=y
```
Comment (smira) [inline kernel/kernel/config-arm64:400]: do we really want to enable it?

File: kernel/kernel/config-arm64 [hunk 2]
```diff
@@ -724,10 +741,11 @@ CONFIG_CRYPTO_AES_ARM64_BS=y
 #
 # General architecture-dependent options
 #
+CONFIG_CRASH_CORE=y
```
Comment (smira) [inline kernel/kernel/config-arm64:744]: same question here

File: kernel/kernel/config-arm64 [hunk 3]
```diff
@@ -883,58 +903,9 @@ CONFIG_MQ_IOSCHED_KYBER=y
 # end of IO Schedulers
 
 CONFIG_PREEMPT_NOTIFIERS=y
+CONFIG_PADATA=y
 CONFIG_ASN1=y
-CONFIG_ARCH_INLINE_SPIN_TRYLOCK=y
```
Comment (smira) [inline kernel/kernel/config-arm64:887]: not sure if we need to remove those inlines?

File: kernel/kernel/config-arm64 (hunk 1 above)
Comment (andrewrynhard) [reply kernel/kernel/config-arm64:400]: No, I got a list of everything added and removed when I ran `make olddefconfig`. I plan on going through that today and fixing up things we don't want to enable.

**Pull request closed without merging**